src/app/calendar/calendar-list/calendar-list.component.ts

```diff
@@ -35,5 +41,9 @@ export class CalendarListComponent implements OnInit, OnChanges {
         return this.selectedDate === currentDate;
       });
     });
+
+    // Transform the selected date using the BulgarianDateFormatPipe
+    this.initialMonth = this.bulgarianDatePipe.transform(this.selectedDate);
+    this.cdr.detectChanges();
```

Comment:
this is very dangerous, why do we need it and don't we have another ways for achieving the wanted result?

Reply:
i forgot to remove it, it was for testing. Removed.